EnglishFilter/sections/general_extensions.txt

```diff
@@ -66,7 +66,7 @@ xxxstreams.me,watchfreexxx.net,playpornfree.org,pandamovie.co,losporn.net,nameth
 xxxparodyhd.net,uptobox.com,uptostream.com,pandamovies.me,pornkino.cc,watchpornfree.info,xxxmoviestream.xyz,mangoporn.co,pandavideo.pw,playpornfree.xyz,xopenload.me,streamporn.pw#%#//scriptlet("abort-on-property-read", "jsUnda")
 !
 ! For scripts, which uses base64
-force-download.es,bro.adca.st,jpost.com,convert-me.com,keezmovies.com#%#window.atob = function() { };
+file-upload.com,force-download.es,bro.adca.st,jpost.com,convert-me.com,keezmovies.com#%#window.atob = function() { };
```

Comment:
better add to existing rule `#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "delete window")`
And chech for unnecessary JS for this site(just list the rules, don't make changes in pull-request).

Reply:
> better add to existing rule `#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "delete window")`

I don't see this rule